src/GitTfs/Commands/Clone.cs

```diff
@@ -129,7 +129,6 @@ public int Run(string tfsUrl, string tfsRepositoryPath, string gitRepositoryPath
 
                     retVal = _initBranch.Run();
                 }
-                _globals.Repository.SetConfig(GitTfsConstants.DisableGitignoreSupport, true);
```

Comment:
Could you explain why it is no more needed?
As we will regress on the behavior introduced by  #1147 to fix #1121

Reply:
> Could you explain why it is no more needed?

Sure. But did you get a chance to read the message for the commit that makes the change...?

> https://github.com/git-tfs/git-tfs/commit/ebfcc6db11001e3ad2019e86cc61021364fbf8b Clone: Avoid disabling gitignore support in configuration
> 
> As "git-tfs" .gitignore support is now disabled by default, avoid needlessly disabling it in configuration file, too, also lowering its exposure at the same time, being a setting which should be used with extreme care, previously getting well informed of its implications - by design causing tracked "TFS" remote and local "Git" repositories getting out of sync, "TFS" files/changes missing on "Git" side.

As that commit states, we already flipped the default .gitignore usage behavior in previous commit (making it disabled by default, so writing configuration setting isn't needed anymore).

So no, we won't regress :) ... and I'm afraid #1147 wasn't solving the whole problem to begin with (being exactly why I've stumbled on this issue myself, and that with >= v0.28 :( For the sake of completeness, here`s the whole story - it's a bit on a longer side, but it does have a happy ending :)

> https://github.com/git-tfs/git-tfs/commit/b5c4464b2b1be6f43599f938064a74cc3577da3c GitTfsRemote: Disable gitignore support by default
> 
> As "git-tfs" uses .gitignore file to support ignoring already tracked remote files (on downloading them from "TFS" remote), it will cause "TFS" and "Git" repositories to get out of sync, ignored "TFS" files missing on "Git" side. This is by design (introduced in 904ae3d, v0.26).
> 
> Unfortunately, being exactly the opposite of how "Git" uses the same file to ignore untracked local files only, not affecting already tracked files downloaded from remote at all, default "git-tfs" behavior (since v0.26) can be quite surprising for the unwary, otherwise seasoned "Git" user, suddenly noticing missing files/changes inside "TFS" cloned "Git" repository (in comparison to tracked "TFS" remote), when one might rightfully expect them to match.
> 
> Issue is partially tackled in 02a557c (v0.28), where .gitignore support is turned off inside configuration after the initial `clone` - but that is not enough, as it only works if `clone` itself was done with at least that version, knowing to turn support off.
> 
> Otherwise, repositories `clone`d with previous "git-tfs" versions would still have .gitignore support "enabled" (as it wouldn't be disabled after `clone`) - and once "git-tfs" is updated to v0.26 or later, .gitignore support would start kicking in, suddenly (and silently) ignoring files on download from "TFS", leaving users wondering what happened[1][2][3].
> 
> While certainly helpful for its purpose of deliberately ignoring specific files (file changes) inside "TFS" remote repository, as it handles the same file in a way that completely differs from what "Git" itself does with it, dropping tracked changes coming from the remote end, .gitignore support inside "git-tfs" could be considered dangerous without explicit user action/configuration, being fully informed of its implications (out of sync remote "TFS" and local "Git" repositories).
> 
> Let's make situation safer overall by disabling "git-tfs" .gitignore support by default (so even when configuration setting is not found), leaving it to the educated user to turn it on inside configuration instead, if desired, explicitly setting "disable-gitignore-support" to "false".
> 
> Note that "enable-gitignore-support" boolean setting might read more intuitively now, but as "disable-gitignore-support" is already known for disabling the previous default, let's keep it for the sake of backward compatibility, at least for now.
> 
> See reported issues #1121 ("git tfs fetch should ignore .gitignore", 2017-11-02)[1], #1140 ("bug caused by .gitignore or 32-bit used on x64?", 2017-12-05)[2] and #1208 ("TFS file incorrectly ignored/missing", 2018-07-08)[3] for additional information.
> 
> [1] #1121
> [2] #1140
> [3] #1208